generators/client/templates/react/src/test/javascript/e2e/modules/account/account.spec.ts.ejs

```diff
@@ -0,0 +1,154 @@
+import SignInPage from '../../page-objects/signin-page';
+import PasswordPage from '../../page-objects/password-page';
+import NavBarPage from '../../page-objects/navbar-page';
+import RegisterPage from '../../page-objects/register-page';
+import { SettingsPage } from '../../page-objects/settings-page';
+
+import { $, browser } from 'protractor';
```

Comment:
Can we put lib imports first followed by local imports, similar to source code we follow